xrviz/display.py

```diff
@@ -61,3 +61,8 @@ def _is_coord(self, name):
             return name + " " + '\U0001F4C8'
         else:
             return name
+
+    @property
+    def kwargs(self):
+        out = {self.panel.select()[0][0].name: self.panel.select()[0][0].value[0]}
```

Comment:
comment here, please" what does `select()` do, and why so many indices?

Reply:
I have simplified this part and added comment.